### ISSUE TITLE
Help me understand your licence model

### ISSUE THREAD

**Issue by ProteanCode** [aimeos/aimeos-laravel#319]
Since the github repository is a MIT-based and the website claims it to be LGPLv3, what is the correct way to stay legal?

After I read this thread on stackexchange:

https://softwareengineering.stackexchange.com/questions/141847/how-does-using-a-lgpl-gem-affect-my-mit-licensed-application/141848#141848

I believe that the Laravel application itself can has a closed source code and the only part of the code I am supposed to expose (on demand) is the one from /vendor directory (or link this repository).

If I would to modify the code of your library, then I should fork the modified repo as public right?

Any code outside your package is not meant to be visible for safety purposes, is your licencing grants that?

**Comment by aimeos** [comment]
Laravel and the Aimeos Laravel package are using the MIT license so you can modify them without making your code public.

The Aimeos core is LGPLv3 licensed which means that you should contribute back changes that are of public interest. Your own Aimeos extension can be or usually is proprietary and you can do any changes you need there (if they are for your project only and not of public interest).

To sum up: Laravel/Aimeos allows you to keep all your own code proprietary and you CAN contribute core changes
